cmd/jaeger/integration/receivers/storagereceiver/config.go

```diff
@@ -0,0 +1,19 @@
+// Copyright (c) 2024 The Jaeger Authors.
```

Comment:
please add a README to this package, e.g.

`storagereceiver` is a fake receiver that creates an artificial stream of traces by:
  * repeatedly querying one of Jaeger storage backends for all traces (by service)
  * tracking new traces / spans and passing them to the next component in the pipeline.